returnn/search.py

```diff
@@ -32,6 +32,7 @@ def __init__(
         model_checkpoint,
         returnn_config,
         *,
+        output_mode="py",
```

Comment:
this should be also added to `hash` function below.

Reply:
the `output_mode` gets added to the return_config param in hash. that should suffice, right!?

Reply:
correct

Reply:
ah yes right.

Reply:
It gets added to self.returnn_config but this is not used in the hash (the original version from the kwargs is used there). So the output_mode currently does not affect the hash.

Created a PR to fix this: https://github.com/rwth-i6/i6_core/pull/62